academic-services/graph/samples-azure-databricks-hindex.md

```diff
@@ -0,0 +1,337 @@
+
+In this section, you create a notebook in Azure Databricks workspace.
+
+1. In the [Azure portal](https://portal.azure.com), go to the Azure Databricks service that you created, and select **Launch Workspace**.
+
+1. On the left, select **Workspace**. From the **Workspace** drop-down, select **Create** > **Notebook**.
+
+    ![Create a notebook in Databricks](media/databricks/databricks-create-notebook.png "Create notebook in Databricks")
+
+1. In the **Create Notebook** dialog box, enter a name for the notebook. Select **Python** as the language.
+
+    ![Provide details for a notebook in Databricks](media/databricks/create-notebook.png "Provide details for a notebook in Databricks")
+
+1. Select **Create**.
+
+## Define configration variables
+
+In this section, you create the first notebook cell and define configration variables.
+
+1. Copy and paste following code block into the first cell.
+
+   ```python
+   AzureStorageAccount = '<AzureStorageAccount>'     # Azure Storage account containing MAG dataset
+   AzureStorageAccessKey = '<AzureStorageAccessKey>' # Access Key of the Azure Storage account
+   MagContainer = '<MagContainer>'                   # The container name in Azure Storage account containing MAG dataset, Usually in forms of mag-yyyy-mm-dd
+
+   MagDir = '/mnt/mag'
+   ```
+
+1. In this code block, replace `AzureStorageAccount`, `AzureStorageAccessKey`, and `MagContainer` placeholder values with the values that you collected while completing the prerequisites of this sample.
+
+   * The `AzureStorageAccount` is the name of your Azure Storage account.
+
+   * The `AzureStorageAccessKey` is the access key of the Azure Storage account.
+
+   * The `MagContainer` is the container name in Azure Storage account containing MAG dataset, Usually in the form of **mag-yyyy-mm-dd**.
+
+1. Press the **SHIFT + ENTER** keys to run the code in this block.
+
+## Mount Azure Storage as a file system of the cluster
+
+In this section, you mount MAG dataset in Azure Storage as a file system of the cluster.
+
+1. Copy and paste the following code block in a new cell.
+
+   ```
+   if (not any(mount.mountPoint == MagDir for mount in dbutils.fs.mounts())):
+     dbutils.fs.mount(
+       source = ('wasbs://%s@%s.blob.core.windows.net' % (MagContainer, AzureStorageAccount)),
+       mount_point = MagDir,
+       extra_configs = {('fs.azure.account.key.%s.blob.core.windows.net' % AzureStorageAccount) : AzureStorageAccessKey})
+
+   dbutils.fs.ls(MagDir)
+   ```
+
+1. Press the **SHIFT + ENTER** keys to run the code in this block.
+
+   You see an output similar to the following snippet:
+
+   ```
+   Out[3]: 
+   [FileInfo(path='dbfs:/mnt/mag/advanced/', name='advanced/', size=0),
+    FileInfo(path='dbfs:/mnt/mag/mag/', name='mag/', size=0),
+    FileInfo(path='dbfs:/mnt/mag/nlp/', name='nlp/', size=0),
+    FileInfo(path='dbfs:/mnt/mag/samples/', name='samples/', size=0)]
+   ``` 
+
+## Extract MAG data files from Azure Storage
```

Comment:
The wording and content here is slightly confusing to me.

Suggestion:
Section title: "Create MAG data frames and temporary views"
Section summary: "In this section you will create data frames and temporary views for several different MAG entity types. These views will be used later on in the tutorial."

Reply:
Also, I'd caveat that some of the cells might take a while to run. I.e. the authors cell took about 10 minutes for me.

Reply:
fixed.